components/automate-cli/cmd/chef-automate/status.go

```diff
@@ -75,6 +119,41 @@ func getStatus() (*api.StatusResponse, error) {
 	return res, nil
 }
 
+func runStatusSummaryCmdFunc(statusSummaryCmdFlags *StatusSummaryCmdFlags) func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		return executeStatusSummary(cmd, args, statusSummaryCmdFlags)
+	}
+}
+
+func executeStatusSummary(cmd *cobra.Command, args []string, statusSummaryCmdFlags *StatusSummaryCmdFlags) error {
+	if isA2HARBFileExist() {
+		infra, err := getAutomateHAInfraDetails()
+		if err != nil {
+			return err
+		}
+		statusSummary := NewStatusSummary(infra, FeStatus{}, BeStatus{}, 10, time.Second, statusSummaryCmdFlags)
+		sshUtil := statusSummary.(*Summary).setSSHConfig()
+
+		err = statusSummary.Run(sshUtil)
+		if err != nil {
+			return err
+		}
+		if isManagedServicesOn() {
+			fe := statusSummary.ShowFEStatus()
+			fmt.Println(fe)
```

Comment:
use `writer` instead of `fmt`
refer other files for `writer`

Reply:
when i use `writer` table is not shown properly